lib/less/components/selects.less

```
@@ -10,6 +10,11 @@
 //  • SELECT MENUS
 //    - Sizes
 //
+
+.d-select-disable {
```

```suggestion
.d-select--disable {
```

We typically use `--` to break up variants

---

Do we need to define this separately if we are using it below as a mixin on the select class when the disabled state is detected?

---

sure, 
`d-select--disable` was switched to mixing
are you ok if keep this mixing on the top of select.less file? in other case could you please mention where it could be moved

---

If the mixin is only being used in a single place `&:disabled` it might be better to moved the styles into the `&:disabled` block

---

done, removed the mixing